src/main/java/org/example/RockPaperScissors/User.java

```diff
@@ -0,0 +1,45 @@
+package org.example.RockPaperScissors;
+
+import java.util.Scanner;
+
+public class User {
+  private Scanner scanner;
+
+   public User() {
+      scanner = new Scanner(System.in);
+   }
+   public RockPaperScissors.Move getMove() {
+       System.out.print("Make a choice: Rock (r), Paper (p), Scissors (s), Water (w), " +
+               "Fire (f), Air (a) or Sponge (g) ? ");
+
+       String userText = scanner.nextLine();
+       userText = userText.toUpperCase();
+       char firstLetter = userText.charAt(0);
+       if (firstLetter == 'R' || firstLetter == 'P' || firstLetter == 'S'|| firstLetter == 'W' ||
```

Comment:
to juz wiecie ze mozna lepiej :D BTW nie wiem czy ten if jest potrzebny nawet, mogliscie w switchu dac default :D